### DISCUSSION TITLE
How to run locally build pytest-html ?

### DISCUSSION THREAD

**Discussion by harmin-parra** [pytest-dev/pytest-html#668] [Q&A] [answered]
Hello

I have cloned your repo and follow the instructions of this page :
https://pytest-html.readthedocs.io/en/latest/development.html

How can I run **pytest** & **pytest-html** using the locally compiled **pytest-html** instead of having python using the already installed **pytest-html** package ?

Thanks

**Comment by BeyondEvil** [comment, marked as answer]
By using "editable install":

In the project where you want to use the local pytest-html you run: `pip install -e <path/to/pytest-html>`
